zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/schema/Schema.scala

```diff
@@ -113,13 +180,19 @@ trait Measurement {
 case class Schema(typeName: String,
                   dimension: Seq[Field],
                   measurement: Seq[Field],
-                  primaryKey: Seq[String],
-                  timeField: String
+                  primaryKey: Seq[Field],
+                  timeField: Option[TimeField]
```

Comment:
why `option`? this field should be always appear. otherwise a lot of logics won't make sense.